### ISSUE TITLE
Class 'Illuminate\Support\Arr' not found

### ISSUE THREAD

**Issue by yooouuri** [laravel/lumen-framework#665]
- Lumen Version: 5.5.*
- PHP Version: PHP 7.1.9-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Sep  2 2017 05:56:43) ( NTS )
- Database Driver & Version: -

### Description:
I am trying to create a new lumen project.

After running `php artisan key:generate` the following error occour

```
[Symfony\Component\Debug\Exception\FatalThrowableError]  
  Class 'Illuminate\Support\Arr' not found
```

### Steps To Reproduce:
`lumen new <name>`
`cd <name>`
`composer install`
`cp .env.example .env`
`php artisan key:generate`

**Comment by ludo237** [comment]
Key generate is not part of lumen default commands, where did you get that?

**Comment by yooouuri** [comment]
@ludo237 

Like laravel, I thought i should generate a key.

When i go to the url of my Lumen website, i got the same error...

**Comment by ludo237** [comment]
You have to generate that, but in other ways anyway..

It's hard to determine the cause of the problem. I tryed the steps above with a fresh lumen installation and I do have the Arr.php in `vendor\illuminate\support`

Have you reference that correctly? Try dumping the composer autoload first

**Comment by martendej** [comment]
I seem to be having the same issue. Ran composer install on a fresh lumen installation, keep getting the 'Illuminate\Support\Arr not found' error. When I dig further in the vendor folder, the Arr.php indeed does not exist. Tried composer dumpautoload, composer clearcache and reinstall, no success. The Arr.php file is just gone.. Any ideas anyone?

**Comment by martendej** [comment]
Ok, I did some research and it seems that some files are not being created when running homestead (I'm using version 5.0.1, Vagrant 1.8.5 & VirtualBox Guest Additions 5.1.12) with the 'NFS' flag in the Homestead.yaml file. After commenting the NFS option and reloading homestead and rerunning composer install all files are created as they should and the project works again. Does anyone know why this is happening? @GrahamCampbell or @taylorotwell maybe?

**Comment by martendej** [comment]
Ok, it seems to be an issue with MacOS High Sierra and the NFS option. Apple fixed it in High Sierra 10.13.2 (beta). [https://github.com/hashicorp/vagrant/issues/8788](This issue covers it) and should clarify some things. I will try updating and I hope the problem is resolved after.
